src/sidescript_readDEM.R

```diff
@@ -1,3 +1,7 @@
+download.file("http://biogeo.ucdavis.edu/data/climate/worldclim/1_4/grid/cur/alt_30s_bil.zip", "data/maps/alt_30s_bil.zip")
```

Comment:
same as above, add if-loop to overcome redundant re-download

Reply:
Same solution here!